packages/schemas/src/types/logto-config.ts

```diff
@@ -77,3 +83,9 @@ export const logtoConfigGuards: LogtoConfigGuard = Object.freeze({
   ...logtoOidcConfigGuard,
   ...logtoTenantConfigGuard,
 });
+
+export const oidcPrivateKeyResponseGuard = oidcPrivateKeyGuard
+  .omit({ value: true })
+  .merge(z.object({ signingKeyAlgorithm: z.nativeEnum(SupportedSigningKeyAlgorithm).optional() }));
```

Comment:
Wondering when should `signingKeyAlgorithm` be undefined?

Reply:
When it's requesting the `oidc.cookieKeys`. The signing algorithm is a JWT thing, and only available to `oidc.privateKeys`

Reply:
cookieKeys should not be oidcPrivateKey type?

Reply:
In database they both share the same object type. 
```ts
type PrivateKey = {
  id: string;
  value: string;
  createdAt: number;
}
```

The `signingKeyAlgorithm` is a calculated value in API response.

Reply:
Discussed offline, and decided to rename the confusing `PrivateKey` to `OidcConfigKey`, as cookie keys are not private key but `OidcConfigKey` can mean both cookie key and private key.